pcdsdevices/mv_interface.py

```diff
@@ -145,6 +147,24 @@ def umvr(self, delta, timeout=None):
         """
         self.umv(delta + self.wm(), timeout=timeout)
 
+    def mv_ginput(self, timeout=None):
```

Comment:
I feel like there should be a `wait` keyword argument here to compliment the `move` call